roles/wildfly_systemd/tasks/systemd.yml

```diff
@@ -107,3 +107,13 @@
       ansible.builtin.include_tasks: service.yml
       vars:
         instance_state: "started"
+
+- name: Ensure policycoreutils-python-utils is installed
+  ansible.builtin.yum:
```

Comment:
replace yum by dnf and maybe move the hardcoded package name into a var in default. The package name may change between OS, so let's be nice and give a chance to the user to make it work in his env :)